server/maintenance_requests.go

```diff
@@ -53,4 +54,14 @@ func (app *App) CreateMaintenanceRequest(c *gin.Context) {
 		return
 	}
 	c.JSON(http.StatusOK, gin.H{"id": req.ID})
-}
\ No newline at end of file
+}
+
+func (app *App) DeleteMaintenanceRequest(c *gin.Context) {
+	id, err := strconv.Atoi(c.Param("id"))
+	deleted_id, err := app.db.DeleteMaintenanceRequest(id)
```

Comment:
Let's return an error here for when `strconv.Atoi` returns an error. The current code will skip over that case. In what case will this code error out? Given that, what status code do you think this should return?